vendor/assets/javascripts/spree/frontend/all.js

```diff
@@ -6,7 +6,5 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require accounting.min
```

Comment:
why we remove this

Reply:
it was not needed as suggested by yasin bhai so he deleted it